README.md

```diff
@@ -1,32 +1,45 @@
 # Requiem
 
-![](https://raw.githubusercontent.com/Ladysnake/Requiem/master/requiem-logo-200x125.png)
+<p align="center">
+    <picture>
+        <source media="(prefers-color-scheme: dark)" srcset="requiem-logo-200x125_light.png" width="250">
+        <img src="requiem-logo-200x125.png" width="250">
+    </picture> 

```

Comment:
Never thought of doing that, that's really cool ! The light version seems hella crispy though.

Reply:
Maybe a grey version with double the resolution would fit better. Curious, do you like the idea of different Logos based on the Color scheme?

Reply:
I quite like the variants, just need to fix the white version's resolution IMO - what tool did you use to swap the colors ?

Reply:
MS Paint 3D